Trigger on job completion (#4870)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/job-scheduler.md

## Scheduler queue

> The scheduler queues a deployment job to be processed when it's triggered to run because of a [set schedule](/docs/deploy/deploy-jobs#schedule-days), [a job completed](/docs/deploy/deploy-jobs#trigger-on-job-completion), an API call, or manual action. 

Reviewer: ```suggestion
The scheduler queues a deployment job to be processed when it's triggered to run by a [set schedule](/docs/deploy/deploy-jobs#schedule-days), [a job completed](/docs/deploy/deploy-jobs#trigger-on-job-completion), an API call, or manual action. 
```
